docs/user_group_management.md

```diff
@@ -0,0 +1,89 @@
+﻿
+  
+
+##  Management in basic authentication mode Guide
+
+  
+In the old version of Openpai,when deployed in basic authentication mode,pai
+didn't provide explicit group information for the admins,thus raising  confusion when managing the platform. In latest version ,pai will provide direct group management .
+
+This document will introduce how to manage user ,group and teamwise storage in Openpai basic authentication mode in the latest version(0.16.0).
+
+  
+
+
+  
+### User Management
+#### Create a new user
+
+  
+
+To add a new user to the cluster,cluster admins needs to do:
+
+  
+
+- create user using webportal or using [rest apis](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/microsoft/pai/master/src/rest-server/docs/swagger.yaml#tag/user)
+- users' username and password must be given when creating a user. 
+
+  
+
+#### Update existing user
+
+  
+
+In basic authentication mode,admin can update the existing users' information directly.
+
+  
+
+- User name ,password and email can be changed in webportal or using [rest apis](https://github.com/microsoft/pai/blob/master/docs/rest-server/API.md).
+
+  
+
+
+- Admin can also update users' grouplist in webportal .A user can belong to  more than one group in basic authentication mode.Admin can add a group or remove a group from users' grouplist.
+#### Delete a user
+- delete user using webportal or using [rest apis](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/microsoft/pai/master/src/rest-server/docs/swagger.yaml#tag/user)
```

Comment:
change link to [deleteUser](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/microsoft/pai/master/src/rest-server/docs/swagger.yaml#operation/deleteUser) api